### PR TITLE
Prepare the release for 2.13.0

### PR DESCRIPTION
**Pull request by tvbeek**
Type: Documentation / release  
Issue: -  
Breaking change: no

### CODE REVIEW

**Review by kylekatarnls — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,21 @@
+pdepend-2.13.0 (2023/02/28)
+==========================
+
+- Added [\#636](https://github.com/pdepend/pdepend/pull/636): Add support for NULL, DEFAULT and NEW as enum case.
+- Added [\#651](https://github.com/pdepend/pdepend/pull/651): Add a follow on mastodon link to the footer on the website.
+- Fixed [\#633](https://github.com/pdepend/pdepend/pull/633): Fix parsing for new with parentheses.
+- Fixed [\#641](https://github.com/pdepend/pdepend/pull/641): Improve UTF8 encoding process.
+- Fixed [\#637](https://github.com/pdepend/pdepend/pull/637): Prevent the use of a not set id value in the AbstractASTType and AbstractASTCallable.
+- Fixed [\#642](https://github.com/pdepend/pdepend/pull/642): Fix [\#638](https://github.com/pdepend/pdepend/issue/638) Unexpected Token with double class modifiers.
+- Changed [\#644](https://github.com/pdepend/pdepend/pull/644): Implement generic visit method.
```
Comment (AJenbo) [inline CHANGELOG.md:10]: Maybe more relevant for the consumers:
```suggestion
- Changed [\#644](https://github.com/pdepend/pdepend/pull/644): Correct collection of Enum and Intersection type information during analysis
```

**Review by AJenbo — APPROVED** [review]
Nice looking forward to this release in particular :)

**Comment by tvbeek** [comment]
> Nice looking forward to this release in particular :)

I understand you have done a lot of work 👍

**Review by AJenbo — APPROVED** [review]

**Comment by AJenbo** [comment]
Yeah, but I'm also really excited for the working enum fixes, we are just about to migrate to 8.1 at work and will start using PHPMD soon as well :)

**Pull request merged**